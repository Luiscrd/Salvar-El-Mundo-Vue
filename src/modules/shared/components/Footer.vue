<template>
  <div class="footer">
    <span class="by-footer">By: LcrPrograming</span>
    <a href="https://www.linkedin.com/in/luis-carballo-rua-3b4763248/" target="_blank">
      <img src="@/assets/in.png" alt="Linked In" class="logo-liked">
    </a>
  </div>
</template>

<script>
export default {
  name: 'Footer',
  props: {
  }
}
</script>

<style scoped>
.footer {
  width: 100%;
  position: relative;
  background-color: #341948;
  color: white;
  padding: 20px 2%;
  display: flex;
  align-items: center;
  justify-content: space-between;
  text-align: center;
}

.by-footer {
  position: relative;
  font-weight: bold;
}

.logo-liked {
  height: 30px;
  transition: 0.5s;
}

.logo-liked:hover {
  opacity: 0.5;
}

@media screen and (max-width: 420px) {
  .footer {
    justify-content: space-between;
    padding: 20px 17px;
  }

  .logo-liked {
    height: 25px;
  }
}
</style>