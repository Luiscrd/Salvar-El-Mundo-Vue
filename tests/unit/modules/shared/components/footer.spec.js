import { shallowMount } from '@vue/test-utils'
import Footer from '@/modules/shared/components/Footer.vue'

const mockStore = {
    getters: jest.fn()
}

jest.mock('vuex', () => ({
    useStore: () => mockStore
}))

describe('App Component', () => {

    let wrapper;

    beforeEach(() => {

        wrapper = shallowMount(Footer)

    });

    test('Debe de hacer mach con el snapshot', () => {

        expect(wrapper.html()).toMatchSnapshot()

    })

})