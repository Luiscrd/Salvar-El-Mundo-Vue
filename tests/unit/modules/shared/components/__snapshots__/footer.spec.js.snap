// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Debe de hacer mach con el snapshot 1`] = `<div class="footer"><span class="by-footer">By: LcrPrograming</span><a href="https://www.linkedin.com/in/luis-carballo-rua-3b4763248/" target="_blank"><img src="" alt="Linked In" class="logo-liked"></a></div>`;
